rebar.config

```diff
@@ -31,8 +31,8 @@
  ]}.
 
 {deps, [
-  {lz4, ".*", {git, "https://github.com/nhs-riak/erlang-lz4", {branch, "nhse-develop-3.4"}}},
-  {zstd, ".*", {git, "https://github.com/nhs-riak/zstd-erlang", {branch, "nhse-develop"}}},
+  {lz4, ".*", {git, "https://github.com/OpenRiak/erlang-lz4", {branch, "openriak-3.4"}}},
+  {zstd, ".*", {git, "https://github.com/OpenRiak/zstd-erlang", {branch, "openriak-3.2"}}},
```

Comment:
just checking... the `3.2` is right here and should not be `3.4`?

Reply:
Yes.  I use openriak-3.2 branch until a change requires the creation of openriak-3.4, and zstd-erlang has not yet required such a change